include/sound/sof/ext_manifest4.h

```diff
@@ -90,6 +90,21 @@ struct sof_man4_segment_desc {
 	uint32_t file_offset;
 } __packed;
 
+/*
+ * bit definition for type in sof_man4_module
+ * uint32_t	load_type : 4;	//MT_BUILTIN, MT_LOADABLE
+ * uint32_t	auto_start : 1;	// single instance created by Module Manager
+ * uint32_t	domain_ll : 1;	// support LL domain
+ * uint32_t	domain_dp : 1;	// support DP domain
+ * uint32_t	lib_code : 1;	// determines if module is place holder for common library code
+ * uint32_t	_rsvd : 24;
```

Comment:
I don't think this table helps at all to understand that bit6 is 'support DP domain' for example, not that it helps to understand what 'DP domain' is...

Note: we already have:
```c
#define SOF_IPC4_MODULE_LL      BIT(5)
```

Reply:
sure